lua/plenary/window/float.lua

```diff
@@ -137,11 +137,13 @@ end
 --@param row_range number | Table:
 --                  If number, then center the window taking up this percentage of the screen.
 --                  If table, first index should be start, second_index should be end
-function win_float.percentage_range_window(col_range, row_range, options)
-  options = tbl.apply_defaults(options, win_float.default_options)
+--@param win_opts Table
+--@param border_opts Table
+function win_float.percentage_range_window(col_range, row_range, win_opts, border_opts)
+  win_opts = tbl.apply_defaults(win_opts, win_float.default_options)
 
-  local win_opts = win_float.default_opts(options)
-  win_opts.relative = "editor"
+  local _win_opts = win_float.default_opts(win_opts)
```

Comment:
I think just call this `win_opts = win_float.default_opts(win_opts)`

I don't like having an underscore prefixed variable that we actually use.

Reply:
Ah, I named this `_win_opts` because we already have a `win_opts` in the parameter list. How about `default_win_opts` here and just leave `win_opts` in the parameter list?